deploy/tm-load-test/tm-load-test.sh

```diff
@@ -0,0 +1,32 @@
+echo "---------------------------------------------------------git checkout------------------------------------------------------------------------------------------------------"
+
+git clone  https://github.com/puneetsingh166/tm-load-test.git
```

Comment:
Why do you clone it here? It either should be cosmos git or onomy fork.

Reply:
We have created a tm-load-test custom client that will broadcast transactions to the Cosmos node. tm-load-test custom client and broadcast code is written in this repo. So we are cloning this repo to apply tm-load-test.